pom.xml

```diff
@@ -53,11 +53,12 @@
          jackson artifacts have both an overall version, and potentially individual versions for artifacts that
          rev hotfix versions independent of the overall version
          -->
-        <jackson.version>2.9.9</jackson.version>
+        <jackson.version>2.9.10</jackson.version>
         <!-- The following property should default to jackson.version unless there's an independent hotfix version.
         If so, then once the next version of jackson is released, it should get reset to the default -->
         <!-- <jackson-databind.version>${jackson.version}</jackson-databind.version> -->
-        <jackson-databind.version>2.9.9.3</jackson-databind.version>
+        <jackson-databind.version>2.9.10.1</jackson-databind.version>
```

Comment:
`jackson-databind.version` was only used here for the dependencyManagement section afaict, so if you're just relying on the bom now then we can just clear this out. In fact, seems like the usage of `jackson.version` in other repos could also just be cleaned up if we are depending on the bom.

Reply:
Good point. I left it in in case a child repo used it, but a quick github search shows it doesn't appear in any poms.

Reply:
Fixed.

Reply:
👍 on cleaning out the other repos of the `jackson.version` property. I've filed ST-2010 to do that, but didn't want to block this one since it's needed for 5.4.